en/docs/learn/lock-user-functionality.md

```diff
@@ -0,0 +1,134 @@
+# Locking/Unlocking Functionality Per User
+
+An admin user can lock and unlock functionalities on a per-user basis. This allows for fine-grained control over functionalities that a particular user is allowed to use. This page guides you through configuring per-user functionality locking for the security-question-based password recovery functionality with WSO2 Identity Server using the OSGi service. 
+
+!!! tip
+    This can also be done using the [User Functionality Management REST APIs](../../develop/user-functionality-mgt-rest-api/). Using the REST API is the recommended approach. 
+
+## Step 1: enable per-user functionality locking
+
+Per-user functionality locking is disabled in WSO2 Identity Server by default. To enable this, add the following configuration in the `deployment.toml` file located in the `<IS_HOME>/repository/conf/` folder. 
+
+```
+[user]
+enable_per_user_functionality_locking="true"
+```
```

Comment:
We decided to enable this by default at the last minute. So, in the product, it is enabled by default. However, the user can toggle the feature on/off with the above configuration in the deployment.toml.